dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/StringConcatenateAssignmentNode.java

```diff
@@ -15,7 +15,11 @@
  * <pre>
  *   <em>variable</em> += <em>expression</em>
  * </pre>
+ *
+ * @deprecated StringConcatenateAssignmentNode is not generated anymore in CFG nodes, and that
+ *     instead an assignment and concatenation is generated.
  */
+@Deprecated
```

Comment:
```suggestion
@Deprecated // 2022-03-22
```

Reply:
Please add the date for the other deprecations, to make it easier to see when we can remove the code.

Reply:
Got it, thanks!

Reply:
@al3xliu Please add this date to the other `@Deprecated` annotations in this PR.

Reply:
Okay!